contribs/gmf/src/directives/disclaimer.js

```diff
@@ -52,8 +56,16 @@ gmf.module.directive('gmfDisclaimer', gmf.disclaimerDirective);
  * @ngdoc controller
  * @ngname GmfDisclaimerController
  */
-gmf.DisclaimerController = function($element, $scope, ngeoDisclaimer,
-     ngeoEventHelper, ngeoLayerHelper) {
+gmf.DisclaimerController = function($element, $scope, ngeoCreatePopup,
+    ngeoDisclaimer, ngeoEventHelper, ngeoLayerHelper) {
+
+  /**
+   * @type {boolean}
+   * @export
+   */
+  this.modal;
+
+  this.modal = this.modal !== undefined ? this.modal : false;
```

Comment:
If you set up the directive with something like `... gmf-disclaimer-modal="ctrl.map"...`, `this.modal` will be a map object. This is why @ger-benjamin prefers to use `this.modal = this.modal === true`.

I would use two different variables. E.g. set up the scope with:

```js
    scope: {
     'modalIn': '<?gmfDisclaimerModal', 
```

And then in your controller:

```js
  /**
   * @type {boolean}
   * @export
   */
  this.modal = this['modaln'] === true;
```

This way you are sure that `this.modal` always has the correct type.